website/source/guides/configuration/lease.html.md

```diff
@@ -0,0 +1,806 @@
+---
+layout: "guides"
+page_title: "Tokens and Leases - Guides"
+sidebar_current: "guides-configuration-lease"
+description: |-
+  Tokens are the core method for authentication within Vault. For every
+  authentication token and dynamic secret, Vault creates a lease
+  containing information such as a duration, renewability, and more.
+  Understanding the lifecycle of lease means understanding the lifecycle of
+  tokens on some sense.
+---
+
+# Tokens and Leases
+
+Almost everything in Vault has an associated lease, and when the lease is
+expired, the secret is revoked. Tokens are not an exception. Every non-root
+token has a time-to-live (TTL) associated with it. When a token expires and it's
+not renewed, the token automatically gets revoked.
+
+## Lease Hierarchy
+
+When a new token or secret is created, it is a child of the creator. If the
+parent is revoked or expires, so do all its children regardless of their own
+leases. A child may be a token, secret, or authentication created by a parent. A
+parent is almost always a **token**.
+
+Suppose a hierarchy exists with respective TTL as follow:
+
+    b519c6aa... (3h)
+        6a2cf3e7... (4h)
+        1d3fd4b2... (1h)
+            794b6f2f... (2h)
+
+In this scenario, the lease ID of `1d3fd4b2..` will expire in an hour. If a
+token or secret with a lease is not renewed before the lease expires, it will be
+revoked by the Vault server. When it's revoked, it takes its child
+(`794b6f2f...`) although the child has one more hour before it expires. Then,
+two hours later, `b519c6aa...` will be revoked and takes its child
+(`6a2cf3e7...`) with it.
+
+
+## Reference Material
+
+- The [Validation](/guides/dynamic-secrets.html#validation) section of the
+[Secret as a Service](/guides/dynamic-secrets.html) guide demonstrated lease
+renewal and revocation
+- [Tokens documentation](/docs/concepts/tokens.html)
+- [Token Auth Backend HTTP API](/api/auth/token/index.html)
+- [Lease, Renew, and Revoke](/docs/concepts/lease.html)
+
+## Estimated Time to Complete
+
+10 minutes
+
+## Personas
+
+The end-to-end scenario described in this guide involves one persona:
+
+- **`admin`** with privileged permissions to create and manage tokens
+
+See the [policy requirements](#policy-requirements) section for details.
+
+## Challenge
+
+Consider the following scenarios:
+
+- Currently, there is no **break glass** procedure available
```

Comment:
...procedure available for revoking access to credentials in the event of a breach